library/src/main/java/com/bumptech/glide/load/resource/bitmap/TransformationUtils.java

```diff
@@ -318,21 +320,31 @@ public static Bitmap rotateImageExif(
     final Matrix matrix = new Matrix();
     initializeMatrixForRotation(exifOrientation, matrix);
 
-    // From Bitmap.createBitmap.
-    final RectF newRect = new RectF(0, 0, inBitmap.getWidth(), inBitmap.getHeight());
-    matrix.mapRect(newRect);
+    Bitmap result;
+    if (VERSION.SDK_INT >= VERSION_CODES.UPSIDE_DOWN_CAKE) {
+      // BitmapPool doesn't support gainmaps (and colorspace), so use Bitmap.createBitmap to apply
+      // the matrix. Bitmap.createBitmap will reuse bitmaps if needed on M+. We could also make this
+      // change across all SDK versions, but we will try with U+ first to derisk this change.
+      result =
+          Bitmap.createBitmap(
+              inBitmap, 0, 0, inBitmap.getWidth(), inBitmap.getHeight(), matrix, true);
```

Comment:
can you argdoc the true parameter, it's unclear without referencing the documentation what this boolean parameter does

Reply:
Done.